CHANGELOG.md

```diff
@@ -5,6 +5,14 @@
 - [#2855](https://github.com/scality/metalk8s/issues/2855) - Bump `containerd`
```

Comment:
NOTE: I didn't added k8s version bump for the moment as I will open another PR to bump k8s to latest 1.18